src/benchmark/backendbench.cc

```diff
@@ -79,8 +75,7 @@ void BackendBenchmark::Run() {
   options.Add<IntOption>(kMaxBatchSizeId, 1, 1024) = 256;
   options.Add<StringOption>(kFenId) = ChessBoard::kStartposFen;
   options.Add<BoolOption>(kClippyId) = false;
-  options.Add<FloatOption>(kClippyThresholdId, 0.0f, 1.0f) = 0.05f;
-  options.Add<FloatOption>(kClippyToleranceId, 0.0f, 1.0f) = 0.03f;
+  options.Add<FloatOption>(kClippyThresholdId, 0.0f, 1.0f) = 0.1f;
```

Comment:
You suggestion to change the default to 0.2 seems reasonable,

Reply:
0.2 would be my best bet for max. playing strength, favoring lower batch sizes. Can be changed later, if someone conducts more tests.